ipa-core/src/net/mod.rs

```diff
@@ -17,6 +22,10 @@ pub use error::Error;
 pub use server::{MpcHelperServer, TracingSpanMaker};
 pub use transport::{HttpShardTransport, HttpTransport};
 
+/// Provides access to IPAs Crypto Provider (AWS Libcrypto).
+static CRYPTO_PROVIDER: Lazy<Arc<CryptoProvider>> =
```

Comment:
something to consider here - I think the cost of maintaining `Lazy` exceeds the cost of calling `default_provider` more than once. `Lazy` does carry a lot of synchronization required to make it safe to call it from multiple threads while here we are just constructing a simple struct that carries a few fields with it. 

Using Lazy make sense when constructing an object is hard and it overweight the cost of synchronization. Here we could've used a function instead

I don't think it is super important here because we only access this when we construct HTTP client and server and we don't do it often, so I am fine leaving it as is